### PR TITLE
update vn.csv

### PR DESCRIPTION
**Pull request by gui7834**

### CODE REVIEW

File: lists/vn.csv [hunk 1]
```diff
@@ -205,3 +205,7 @@ https://cpj.org/asia/vietnam/,HUMR,Human Rights Issues,2019-09-25,OONI,
 http://truongduynhat.org/,POLR,Political Criticism,2019-09-25,OONI,
 https://www.hrw.org/legacy/pub/2006/manifesto_040606.pdf,POLR,Political Criticism,2019-09-25,OONI,
 https://ncov.moh.gov.vn/,PUBH,Public Health,2020-04-01,citizenlab,coronavirus
+https://sbs.com.au/,NEWS,News Media,2020-12-14,OONI,
+https://amnesty.org/,HUMR,Human Rights Issues,2020-12-14,OONI,
```
Comment (hellais) [inline lists/vn.csv:209]: This is already included in the global test list

File: lists/vn.csv [hunk 2]
```diff
@@ -205,3 +205,7 @@ https://cpj.org/asia/vietnam/,HUMR,Human Rights Issues,2019-09-25,OONI,
 http://truongduynhat.org/,POLR,Political Criticism,2019-09-25,OONI,
 https://www.hrw.org/legacy/pub/2006/manifesto_040606.pdf,POLR,Political Criticism,2019-09-25,OONI,
 https://ncov.moh.gov.vn/,PUBH,Public Health,2020-04-01,citizenlab,coronavirus
+https://sbs.com.au/,NEWS,News Media,2020-12-14,OONI,
```
Comment (hellais) [inline lists/vn.csv:208]: This is also in the global list already

File: lists/vn.csv [hunk 3]
```diff
@@ -205,3 +205,7 @@ https://cpj.org/asia/vietnam/,HUMR,Human Rights Issues,2019-09-25,OONI,
...
+https://amnesty.org/,HUMR,Human Rights Issues,2020-12-14,OONI,
+https://amnestyusa.org/,HUMR,Human Rights Issues,2020-12-14,OONI,
```
Comment (hellais) [inline lists/vn.csv:210]: Already present in the global list

**Comment by hellais** [comment]
Thanks for opening this PR, could you update the PR to only include the bitcoinvn website as the others are already provided by the global test-list?

**Comment by agrabeli** [comment]
@hellais I have removed the URLs that are already included in the global test list.

**Comment by agrabeli** [comment]
Thank you @gui7834

**Pull request merged**